.pre-commit-config.yaml

```diff
@@ -3,13 +3,19 @@ repos:
     hooks:
       - id: ruff
         name: ruff
-        entry: ruff
-        args: ["--fix", "--show-source"]
-        files: "numpyro/.*|tests/.*"
+        entry: ruff check
+        args: ["--fix", "--output-format=full"]
+        files: "numpyro/.*|tests/.*|examples/*.py|notebooks/*.ipynb"
```

Comment:
just curious: would this include subfolders, like notebooks/source or examples/cvae_flax?

Reply:
Now that you mention this, I'll actually try to remove this so that it can target all relevant files as in the `make` commands 👌

Reply:
ok! 😅 After some manual tests and edge cases I think https://github.com/pyro-ppl/numpyro/pull/1795/commits/44595d024e8a7cf129d9003034c0d51706540839 is what we want :D

Reply:
thanks!